llvm/test/Transforms/LoopVectorize/AArch64/force-inst-cost-invalid.ll

```diff
@@ -0,0 +1,24 @@
+; REQUIRES: asserts
+; RUN: opt < %s -passes=loop-vectorize -force-target-instruction-cost=1 -debug-only=loop-vectorize -S -disable-output 2>&1 | FileCheck %s
```

Comment:
Can you move the test to `llvm/test/Transforms/LoopVectorize/AArch64/force-target-instruction-cost.ll`, which aloready has similar tests

Reply:
I've moved the test into `force-target-instruction-cost.ll` and changed the flag in that test to `-force-target-instruction-cost=100` as suggested.